### ISSUE TITLE
Not working: "Failed to publish note to Obsius" error

### ISSUE THREAD

**Issue by bnleez1** [jonstodle/obsius-obsidian-plugin#16]
Receiving, "Failed to publish note to Obsius" error...

**Comment by pamela-drouin** [comment]
I'm also getting this error (this is the first time I've tried using the plug-in).

**Comment by ajpelu** [comment]
hi, I got the same error. Any ideas?

**Comment by nrkimv6** [comment]
I got same error and obsiusWrapper.createPost returns null value T_T

**Comment by mcudrio** [comment]
Hi, same for me to, not working.
Obsidian 1.5.3 and installer 1.5.3
Thank you for your help.

**Comment by TangentFoxy** [comment]
I had this installed for a while, but hadn't actually tried it until just now, where I got the same meaningless error. :\

**Comment by jonstodle** [comment]
Hello everyone! I'm sorry I've not responded. Long story short, I had some stuff I needed to work out.

Anyway! I've just published an update to the plugin, and it should now work as before!

Head to Obsidian settings > Community plugins > Current plugins > Update all. That should present an option to update to version `1.3.0` which has the fix.

Thank you for using Obsius and thank you for reporting the issue.
